packages/analytics/analytics-chart/src/components/VueJsonCsv.vue

```diff
@@ -118,22 +99,19 @@ const exportableData = computed(() => {
 
 const labelsFunctionGenerator = (): any => {
   const labels: any = props.labels
-  if (
-    !isType(labels, ValidType.Undefined) &&
-      !isType(labels, ValidType.Function) &&
-      !isType(labels, ValidType.Object)
-  ) {
+
+  if (![ValidType.Undefined, ValidType.Function, ValidType.Object].includes(typeof labels as ValidType)) {
     throw new Error('Labels needs to be a function(value,key) or object.')
   }
 
-  if (isType(labels, ValidType.Function)) {
+  if (typeof labels as ValidType === ValidType.Function) {
```

Comment:
Rather than doing all these changes, if this works, then you can just keep the original function and update it to be 

```
const isType = (value: any, type: ValidType) => {
  return typeof value as ValidType === type
}
```